cpp/src/gandiva/cache.cc

```diff
@@ -20,10 +20,10 @@
 
 namespace gandiva {
 
-static const int DEFAULT_CACHE_SIZE = 500;
+static const size_t DEFAULT_CACHE_SIZE = 128 * 1024 * 1024;  // 256 MiB
 
-int GetCapacity() {
-  int capacity;
+size_t GetCapacity() {
+  size_t capacity;
   const char* env_cache_size = std::getenv("GANDIVA_CACHE_SIZE");
```

Comment:
rename the constant as GANDIVA_CACHE_SIZE_BYTES (otherwise it will pick up the earlier set value on existing machines)

Reply:
Makes sense, changed to it